File: YolHizProblemiHesaplama.java
import java.util.Scanner;

public class YolHizProblemiHesaplama {
    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);
        System.out.printf("Mesafeyi Giriniz :");
        int mesafe = sc.nextInt();
        System.out.printf("Hizi Giriniz :");
        int hiz = sc.nextInt();
        int sure = mesafe / hiz;
        System.out.printf("Sure %d saattir :", sure);
    }
}

File: YolHizProblemiHesaplama.cs
using System;

class Program
{
    static void Main(string[] args)
    {
        float mesafe, hiz, sure;

        //Mesafeyi alıyoruz.
        Console.WriteLine("Mesafeyi Giriniz (km):");
        mesafe = float.Parse(Console.ReadLine());

        //Hızı alıyoruz.
        Console.WriteLine("Hızı Giriniz (km/h):");
        hiz = float.Parse(Console.ReadLine());

        //Süreyi hesaplıyoruz.
        sure = mesafe / hiz;

        //Ne kadar sürede gidileceğini ekrana yazdırıyoruz.
        Console.WriteLine($"Süre: {sure} saat");
    }
}
